app/pages/Hive/Home.tsx

```diff
@@ -35,15 +31,124 @@ const SBT_TOKENS: SBTToken[] = [
+interface MultiplierRule {
+	title: string;
+	badge: string;
+	badgeColor: string;
+	description: string;
+	items: {
+		id: string;
+		label: string;
+		value: string;
+		description: string;
+	}[];
+}
+
+const MULTIPLIER_RULES: MultiplierRule[] = [
+	{
+		title: "Global Multipliers",
+		badge: "Stackable",
+		badgeColor: "bg-success/10 text-success border-success-500/20",
+		description: "These bonuses add up on top of your base score.",
+		items: [
+			{
+				id: "early_bee",
+				label: "Early Bee",
+				value: "+20%",
+				description: "First 2,000 eligible wallets.",
+			},
+		],
+	},
+	{
+		title: "Beeliever Status",
+		badge: "Highest Applies",
+		badgeColor: "bg-primary/10 text-primary border-primary/20",
+		description: "Based on the rarity of your NFT. These do not stack (highest takes priority).",
+		items: [
+			{
+				id: "nft_normal",
+				label: "Standard Holder",
+				value: "+15%",
+				description: "Hold a Beeliever NFT",
+			},
+			{ id: "nft_mythic", label: "Mythic Holder", value: "+30%", description: "Hold a Mythic NFT" },
+		],
+	},
+	{
+		title: "Whale Bonus",
+		badge: "Highest Applies",
+		badgeColor: "bg-info/10 text-info border-info/20",
+		description: "Based on the quantity of NFTs held. These do not stack (highest takes priority).",
+		items: [
+			{ id: "qty_5", label: "Hive 5", value: "+5%", description: "Hold ≥ 5 NFTs" },
+			{ id: "qty_10", label: "Hive 10", value: "+10%", description: "Hold ≥ 10 NFTs" },
+			{ id: "qty_20", label: "Hive 20", value: "+15%", description: "Hold ≥ 20 NFTs" },
+		],
+	},
 ];
 
+const MultiplierGroup = ({ group }: { group: MultiplierRule }) => {
+	const [isOpen, setIsOpen] = useState<boolean>(false);
+
+	return (
+		<div className="overflow-hidden rounded-xl border">
+			<button
+				onClick={() => setIsOpen(!isOpen)}
+				className="hover:bg-base-100/50 flex w-full items-center justify-between p-4 transition-colors"
+			>
+				<div className="flex items-center gap-3">
+					<h3 className="text-lg font-semibold">{group.title}</h3>
+					<span
+						className={`rounded border px-2 py-0.5 text-[10px] font-bold uppercase ${group.badgeColor}`}
+					>
+						{group.badge}
+					</span>
+				</div>
+				{isOpen ? <ChevronUp className="h-5 w-5" /> : <ChevronDown className="h-5 w-5" />}
+			</button>
+
+			<div
+				className={`overflow-hidden transition-all duration-300 ease-in-out ${isOpen ? "max-h-[500px] opacity-100" : "max-h-0 opacity-0"}`}
+			>
+				<div className="border-base-100/50 border-t p-4 pt-0">
+					<p className="text-base-content/70 mt-4 mb-4 flex items-start gap-2 text-sm">
+						<Info className="mt-0.5 h-4 w-4 shrink-0" />
+						{group.description}
+						{group.badge === "Highest Applies" && (
+							<span className="text-primary-foreground ml-1">
+								*These do not stack, the highest takes priority.
+							</span>
+						)}
+					</p>
+
+					<div className="overflow-x-auto">
+						<table className="w-full text-left text-sm">
```

Comment:
why do we use again custom styles, rather than using daisy UI? - table and many other places!

Reply:
This was in the design. I directly used the in built component now.